cmd/create/cluster/cmd.go

```diff
@@ -1649,6 +1659,44 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	if enableProxy && interactive.Enabled() {
+		reporter.Warnf("use \"*\" with double quotes to bypass proxy for all destinations")
```

Comment:
Since this is for interactive mode, I think we can use the `interactive.PrintHelp` here, as it flows better with the rest of the interactive prompts (it's not really a warning).

Reply:
@vkareh do you see another way to show this?
```
? HTTPS proxy (optional): 
? use "*" with double quotes to bypass proxy for all destinations
? No proxy (optional): [? for help] 
```
I could not place the interactive.PrintHelp elsewhere as it returns an error that needs to be checked

Reply:
hmm... I just compiled this and the reality is that for interactive mode we don't need the double-quotes. In fact, entering double quotes gave me an error, but a naked asterisk worked:
```
? Use cluster-wide proxy (optional): Yes
? HTTP proxy (optional): http://localhost
? HTTPS proxy (optional): 
? No proxy: use "*" with double quotes to bypass proxy for all destinations
X Sorry, your reply was invalid: expected a valid user no-proxy value: '"*"' matching ^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])$|^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\/(3[0-2]|[1-2][0-9]|[0-9]))$|^.?(([a-zA-Z]|[a-zA-Z][a-zA-Z0-9-]*[a-zA-Z0-9])\.)*([A-Za-z]|[A-Za-z][A-Za-z0-9-]*[A-Za-z0-9])$|^\*$|^""$
? No proxy (optional): *
? Additional trust bundle file path (optional): 
```
I think just remove this warning entirely

Reply:
So we are left with NON interactive mode where   --no-proxy *    still converts it to assets.  I'm in favor of adding a description to the 'rosa create cluster --help' no-proxy field and letting the user figure this out. WDYT? The only other way is to compare to the first directory name in the project, but it's not so easy, as the OS call returns also the hidden directories so the first directory is not 'assets' but a hidden one.

Reply:
> I'm in favor of adding a description to the 'rosa create cluster --help' no-proxy field and letting the user figure this out.

I think we have no choice, let's do that.

> The only other way is to compare to the first directory name in the project, but it's not so easy, as the OS call returns also the hidden directories so the first directory is not 'assets' but a hidden one.

That's opening a can of worms and I don't think we should pursue that.

Reply:
so what's your solution then? Reading all directories and removing the hidden directories and files from the list, in order to get the first one?

Reply:
uh, no, we don't want to do that.

What you said in the previous comment: Let's add a description to the `rosa create cluster --help` no-proxy field and let the user figure it out.